translate/rus.json

```diff
@@ -41,7 +41,7 @@
 "background reset":"сброс фона",
 "base border":"бордюр основания",
 "base color":"цвет основания",
-"base mix":"смесь основания",
+"base mix":"цветность основания",
```

Comment:
This input changes how much color we apply to the base of an arrow which shows engines' moves. I see what have you tried to mean with this, but this is wrong.  We usually mark this characteristic with the word 'цвет' but since it is used in a more convenient way, I had to come up with this word to replace it.